### DISCUSSION TITLE
How can we add new concourse resource in the resource catalog

### DISCUSSION THREAD

**Discussion by shyamz-22** [concourse/concourse#5399] [General]
I have a [newrelic resource](https://github.com/shyamz-22/newrelic-deployment-resource) for adding deployment markers, What is the procedure for it to be added in to the https://resource-types.concourse-ci.org/

Thanks
Syamala

**Comment by jomsie** [comment]
Hi @shyamz-22!

The current method to get your resource onto the resource types catalog is to make a pull request to this repo: https://github.com/concourse/resource-types.

You can see from the [readme](https://github.com/concourse/resource-types/blob/master/README.md) what steps are needed to get the PR approved by either @YoussB or myself.

Let me know if you have any further issues, and thanks for contributing!

James

**Reply by shyamz-22** [reply]
Hi James,

Thank you for the swift reply. I have created a PR here https://github.com/concourse/resource-types/pull/37.
